marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/EntitySearchControllerTest.java

```diff
@@ -45,6 +45,9 @@ void testCRUDOnSavedQuery() throws Exception {
             "  }\n" +
             "}";
 
+        setTestUserRoles("hub-central-saved-query-writer", "hub-central-saved-query-reader");
+        loginAsTestUser();
```

Comment:
No need to change, but you can do this in one method - loginAsTestUserWithRoles.